### PR TITLE
feat: Add new email templates

### PR DESCRIPTION
**Pull request by AdityaHegde**
Updates various email templates to match newer design.

### CODE REVIEW

File: admin/server/billing.go [hunk 1]
```diff
@@ -193,12 +194,26 @@ func (s *Server) UpdateBillingSubscription(ctx context.Context, req *adminv1.Upd
...
+
+		if plan.PlanType == billing.TeamPlanType {
+			// special handling for team plan to send custom email
+			err = s.admin.Email.SendTeamPlanStarted(&email.TeamPlan{
+				ToEmail:          org.BillingEmail,
+				ToName:           org.Name,
+				OrgName:          org.Name,
+				FrontendURL:      s.admin.URLs.Frontend(),
+				PlanName:         plan.DisplayName,
+				BillingStartDate: sub.CurrentBillingCycleStartDate, // TODO: should this be end date + 1?
```
Comment (pjain1) [inline admin/server/billing.go:206]: depends on what we want if this changed at any day we want to show current day or day+1 @ericokuma can confirm ?

File: proto/rill/admin/v1/api.proto [hunk 2]
```diff
@@ -2409,9 +2409,18 @@ message AlertOptions {
   string web_open_state = 12; // Annotation for the base64-encoded UI state to open for the report.
 }
 
+enum BillingPlanType {
```
Comment (pjain1) [inline proto/rill/admin/v1/api.proto:2412]: what does front end use it for ? similar comment as other

File: web-admin/src/features/billing/plans/utils.ts [hunk 3]
```diff
@@ -19,15 +22,15 @@ export function formatUsageVsQuota(
 }
 
 export function isTrialPlan(plan: V1BillingPlan) {
```
[the remark this answers is not in the document]
Comment (AdityaHegde) [reply web-admin/src/features/billing/plans/utils.ts:24]: Ya we decide what type of interactions to add in the billing page based on the plan type. We mainly use `isTeamPlan` and `isPOCPlan`

File: admin/jobs/river/biller_event_handlers.go [hunk 4]
```diff
@@ -162,11 +162,12 @@ func (w *PaymentSuccessWorker) Work(ctx context.Context, job *river.Job[PaymentS
...
 	err = w.admin.Email.SendInvoicePaymentSuccess(&email.InvoicePaymentSuccess{
-		ToEmail:  org.BillingEmail,
-		ToName:   org.Name,
-		OrgName:  org.Name,
-		Currency: failedInvoice.Currency,
-		Amount:   failedInvoice.Amount,
+		ToEmail:          org.BillingEmail,
+		ToName:           org.Name,
+		OrgName:          org.Name,
+		BillingStartDate: time.Time{}, // TODO: which will this be?
```
[the remark this answers is not in the document]
Comment (pjain1) [reply admin/jobs/river/biller_event_handlers.go]: Is `BillingStartDate` the start date of invoice ? Don't think its available here, here's what is available - https://github.com/orbcorp/orb-go/blob/main/invoice.go#L203

File: web-admin/src/features/billing/plans/utils.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (pjain1) [reply web-admin/src/features/billing/plans/utils.ts:24]: ok

File: admin/server/billing.go (hunk 1 above)
Comment (AdityaHegde) [reply admin/server/billing.go]: Updated to use end date.

File: admin/jobs/river/biller_event_handlers.go (hunk 4 above)
[the remark this answers is not in the document]
Comment (AdityaHegde) [reply admin/jobs/river/biller_event_handlers.go]: Removed this

**Review by pjain1 — APPROVED** [review]

**Pull request merged**